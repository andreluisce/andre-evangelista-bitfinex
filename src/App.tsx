import { Provider } from 'react-redux';
import React from 'react';
import ThemeProvider from './shared/theme';

import store from './shared/state';

import OrderBook from './components/order-book-widget';

function App() {
    return (
        <Provider store={store}>
            <ThemeProvider>
                <OrderBook />
            </ThemeProvider>
        </Provider>
    );
}

export default App;
