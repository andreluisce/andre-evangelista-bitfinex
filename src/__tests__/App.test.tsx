import { render, screen, waitFor } from '@testing-library/react';

import App from '../App';

describe('Render complete app', () => {
    it('should successfully render the requested response"', async () => {
        const { container } = render(<App />);

        expect(container).toMatchSnapshot();
    });
});
