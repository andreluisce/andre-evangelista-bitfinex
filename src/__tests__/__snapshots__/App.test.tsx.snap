// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render complete app should successfully render the requested response" 1`] = `
<div>
   
  <button>
    Disconnect
  </button>
  <section
    class="css-13xtug4"
  >
    <div
      class="css-wndu8q"
    >
      <div
        class="css-1h5rp2e-OrderBookTable"
      >
        <div>
          Count
        </div>
        <div>
          Price
        </div>
        <div>
          Amount
        </div>
      </div>
    </div>
    <div
      class="css-wndu8q"
    >
      <div
        class="css-1h5rp2e-OrderBookTable"
      >
        <div>
          Count
        </div>
        <div>
          Price
        </div>
        <div>
          Amount
        </div>
      </div>
    </div>
  </section>
   
</div>
`;
